pub-critic/src/app/components/sideMenu/SideMenu.module.scss

```diff
@@ -0,0 +1,68 @@
+.SideMenu {
+    width: 100%;
+    height: 100vh;
+    position: fixed;
+    top: 0;
+    display: flex;
+    flex-direction: row;
+}
+.Backdrop {
+    width: 70%;
+    height: 100%;
+    background-color: black;
+    transform: translateX(-300%);
+    position: fixed;
+    top: 0;
+    left: 0;
+    opacity: 0.5;
+    z-index: 99;
+    transition: transform 0.3s ease-in-out;
+    @media (max-width: 768px) {
+        width: 50%;
+    }
+}
+.SideMenuContent {
+    width: 30%;
+    height: 100%;
+    background-color: #151515;
+    position: fixed;
+    top: 0;
+    right: 0;
+    z-index: 100;
+    transform: translateX(100%);
+    transition: transform 0.3s ease-in-out;
+    @media (max-width: 768px) {
+        width: 50%;
+    }
+    .Links {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+        height: 100%;
+        .Link {
+            font-size: 1.5vw;
+            font-weight: 500;
+            color: white;
+            width: 100%;
+            text-align: center;
+            transition:
+                color 0.3s ease-in-out,
+                background-color 0.3s ease-in-out;
+            &:hover {
+                color: #9ca3af;
+                cursor: pointer;
+                background-color: #ccc;
+            }
+            margin-bottom: 20px;
+            @media (max-width: 768px) {
+                font-size: 3.5vw;
+            }
+        }
+    }
+}
+
+.Active {
+    transform: translateX(0);
+}
```

Comment:
You might need Prettier for the formatting

Reply:
I setup a wrong formatter before, now everything will be formatted in prettier :white_check_mark:

Reply:
I will commit the fixes to the repo when I fix everything and request a re-review